CHANGELOG.md

```diff
@@ -42,7 +43,7 @@
 ### Minor
 
 - Button: Fix transparent button on blue background (#316)
-- Flyout: Add new prop `shouldFocus` to override focus on open behavior. *Has codemod* (#325)
```

Comment:
Oops, this is an accident. My VS Code insists on formatting to use _ instead of *.